contrib/windows/appveyor_build.sh

```diff
@@ -198,6 +196,7 @@ if [ -n "$USEMSVC" ]; then
 else
   # Use BinaryBuilder
   echo 'USE_BINARYBUILDER_LLVM = 1' >> Make.user
+  echo 'USE_BINARYBUILDER_OPENBLAS = 1' >> Make.user
   echo 'BINARYBUILDER_LLVM_ASSERTS = 1' >> Make.user
   echo 'override DEP_LIBS += llvm openlibm' >> Make.user
```

Comment:
`blas` needs to be added here. Right now we are still pulling it from `latest`

Reply:
What do you mean pulling it from `latest`?

Reply:
I see; so this is like a hacky way of working around `USE_SYSTEM_OPENBLAS`?

Reply:
Yeah... we pull in dependencies from the nightly build, then pretend that they are system libraries, but for the binarybuilder we then need too add those back... Don't ask me why I just work here... ;)